### ISSUE TITLE
AppImage takes `-compress dwaa` for EXR, but outputs an image with no compression

### ISSUE THREAD

**Issue by HugLifeTiZ** [ImageMagick/ImageMagick#7243]
### ImageMagick version

7.1.1-29

### Operating system

Linux

### Operating system, version and so on

KDE Neon 5.27

### Description

I want to recompress some HDRIs that I'm downloading from Poly Haven to save space. They come with PIZ compression. ZIP compression works and improves their storage a little bit--enough to be worth using for now--but DWAA would save even more. However, attempting to use it results in an uncompressed file.

### Steps to Reproduce

```
❯ magick venetian_crossroads_16k.exr -compress zip venetian_crossroads_16k_zip.exr; magick venetian_crossroads_16k.exr -compress dwaa venetian_crossroads_16k_dwaa.exr; du -sh venetian_crossroads*
1.1G    venetian_crossroads_16k_dwaa.exr
253M    venetian_crossroads_16k.exr
210M    venetian_crossroads_16k_zip.exr
❯ file venetian_crossroads*
venetian_crossroads_16k_dwaa.exr: OpenEXR image data, version 2, storage: scanline, compression: none, dataWindow: (0 0)-(16383 8191), displayWindow: (0 0)-(16383 8191), lineOrder: increasing y
venetian_crossroads_16k.exr:      OpenEXR image data, version 2, storage: scanline, compression: piz, dataWindow: (0 0)-(16383 8191), displayWindow: (0 0)-(16383 8191), lineOrder: increasing y
venetian_crossroads_16k_zip.exr:  OpenEXR image data, version 2, storage: scanline, compression: zip, dataWindow: (0 0)-(16383 8191), displayWindow: (0 0)-(16383 8191), lineOrder: increasing y
```

### Images

You can grab the image I tested with here: https://polyhaven.com/a/venetian_crossroads

**Comment by snibgo** [comment]
What version of OpenEXR does your installation of IM use? For example, I have:
```
D:\temp>%IMG7%magick -list format | grep -i exr
      EXR  rw-   High Dynamic-range (HDR) (OpenEXR 3.2.1)
```
... and this seems to have successful DWAA and DWAB compression for EXR.

**Comment by HugLifeTiZ** [comment]
```
❯ magick -list format | grep -i exr
      EXR  rw-   High Dynamic-range (HDR) (OpenEXR 2.3.0)
```

That seems... old. Maybe whoever or whatever builds the AppImage should revisit the dependencies that get bundled with it. In the meantime, I could see if there are any Flatpak applications that bundle ImageMagick, and see if I can't find a way to jerry-rig executing that instead.

**Comment by dlemstra** [comment]
The AppImage uses the dependencies that are available on the build image (Ubuntu 20.04). We will not be building the latest dependencies ourselves. Using an extra apt repository could but an option so we are open to pull requests with those kind of changes.
